### DISCUSSION TITLE
podman for Windows on ARM support

### DISCUSSION THREAD

**Discussion by DawgTool** [containers/podman#21212] [General]
HI All,
I see that ARM64 support for Windows is part of the Podman Desktop options, however the podman.exe version that is installed as part of the initial setup process is x64. Is there are version of podman (and podman-compose) that is native to ARM64 on Windows?
Thanks

**Comment by Luap99** [comment]
@n1hility PTAL

**Comment by itpropro** [comment]
Any updates on this? Will there be a Windows ARM64 release?

**Comment by n1hility** [comment]
Windows arm is supported. The executable is x86-64, but it is transpiled on run and it communicates with an arm Linux distro

**Reply by itpropro** [reply]
So it is running natively like Docker for Windows on ARM and not through Prism?

**Comment by Luap99** [comment]
Tracked in https://github.com/containers/podman/issues/23121

**Reply by itpropro** [reply]
Thanks, that is what I was looking for!
